proposals/VK_EXT_map_memory_placed.adoc

```diff
@@ -0,0 +1,178 @@
+// Copyright 2022-2023 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+# VK_EXT_map_memory_placed
+:toc: left
+:refpage: https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/
+:sectnums:
+
+This document proposes adding support client-controlled virtual address
+placement of `VkDeviceMemory` maps.
+
+## Problem Statement
+
+There are certain cases in which it can be useful to directly control the
+address used by a memory map operation.  One example of this is in
+emulation environments when the application under emulation is 32-bit but
+the host userspace is 64-bit.  In order to avoid additional copies or
+address translation, the emulator can choose an address in the lower 32
+bits of the address space and request that the map be placed there.
```

Comment:
Somewhat nitpicky since we aren't running the reflow tools on proposals (maybe we should?), but for the sake of future diffing it would be nice to have sentences start on new lines.

Reply:
Done. I double-checked the appendix as well.